packages/tpetra/core/src/Tpetra_CrsGraph_decl.hpp

```diff
@@ -87,6 +87,17 @@ namespace Tpetra {
              const Teuchos::RCP<typename OutputCrsGraphType::node_type> nodeOut,
              const Teuchos::RCP<Teuchos::ParameterList>& params = Teuchos::null);
     };  // class CrsGraphCopier
+
+    template<class LO, class GO, class NT>
+    void
+    unpackCrsGraphAndCombine(
```

Comment:
What's going on here?  Was this declaration not here already?

Reply:
@mhoemmen that got picked up when I merged develop into my branch... I had a merge conflict so that's why it's being picked up... I just picked what came in from the `develop` branch.